src/org/labkey/test/pages/core/login/LoginConfigurePage.java

```diff
@@ -86,25 +159,51 @@ protected ElementCache newElementCache()
 
     protected class ElementCache extends LabKeyPage.ElementCache
     {
-        WebElement primaryConfigsPanel()
+        WebElement globalSettingsPanel()
+        {
+            return Locator.tagWithClass("div", "panel-default")
+                    .withChild(Locator.tagWithClass("div", "panel-heading")
+                            .withChild(Locator.tag("span").withText("Global Settings")))
+                    .waitForElement(this, WAIT_FOR_JAVASCRIPT);
+        }
+
+        Locator checkBoxLoc(String label)
+        {
+            return Locator.tagWithClass("div", "global-settings__text-row").containing(label)
+                    .child(Locator.tagWithClass("span", "clickable"));
+        }
+
+        WebElement selfSignupCheckBox = checkBoxLoc("Allow self sign up").findWhenNeeded(globalSettingsPanel())
+                .withTimeout(WAIT_FOR_JAVASCRIPT);
+        WebElement allowUserEmailEditCheckbox = checkBoxLoc("Allow users to edit their own email addresses")
+                .findWhenNeeded(globalSettingsPanel()).withTimeout(WAIT_FOR_JAVASCRIPT);
+        WebElement autoCreateCheckBox = checkBoxLoc("Auto-create authenticated users")
+                .findWhenNeeded(globalSettingsPanel())
+                .withTimeout(WAIT_FOR_JAVASCRIPT);
```

Comment:
Does the `SvgCheckbox` component not work for these ones?

Reply:
What happened here... is I didn't refactor to use them here.  Good catch